lib/ffmpeg/vaapi/vpp.py

```diff
@@ -42,6 +42,25 @@ def gen_vpp_opts(self):
     elif self.vpp_op in ["overlay"]:
       vpfilter.append("[1:v]hwupload[v1];")
       vpfilter[-1] += "[0:v][v1]overlay_vaapi=alpha={a}".format(a = mapRange(self.alpha, [0.0, 255.0], [0.0, 1.0]))
+    elif self.vpp_op in ["drawbox"]:
+      vpopts = []
+      vpfilter.append("drawbox_vaapi")
+      if hasattr(self, "boxw"):
+        vpopts.append(f"width={self.boxw}")
+      if hasattr(self, "boxh"):
+        vpopts.append(f"height={self.boxh}")
+      if hasattr(self, "x"):
+        vpopts.append(f"x={self.x}")
+      if hasattr(self, "y"):
+        vpopts.append(f"y={self.y}")
+      if hasattr(self, "color"):
+        vpopts.append(f"color={self.color}")
+      if hasattr(self, "t"):
+        vpopts.append(f"t={self.t}")
+      if hasattr(self, "replace"):
+        vpopts.append(f"t={self.replace}")
```

Comment:
Both use the same ffmpeg option `t`?

Reply:
Thanks for catching this typo, fixed in the new version.